<ui version="4.0" >
 <class>ConfigurarImpresoraClass</class>
 <widget class="QWidget" name="ConfigurarImpresoraClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>349</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>349</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Configurar Impresora</string>
  </property>
  <widget class="QWidget" name="gridLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="1" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Puerto de la impresora</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QComboBox" name="comboBox" />
    </item>
    <item row="0" column="0" >
     <widget class="QCheckBox" name="checkBox" >
      <property name="text" >
       <string>Habilitar configuraci√≥n manual</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
      <property name="tristate" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1" >
     <widget class="QDialogButtonBox" name="buttonBox" >
      <property name="standardButtons" >
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QSpinBox" name="spinBox" >
      <property name="maximum" >
       <number>999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Largo</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Ancho</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" >
     <widget class="QSpinBox" name="spinBox_2" >
      <property name="maximum" >
       <number>999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>72</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel" >
     <x>214</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>102</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel" >
     <x>77</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>98</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel" >
     <x>197</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>50</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel" >
     <x>35</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>177</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel" >
     <x>177</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>136</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel" >
     <x>222</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigurarImpresoraClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>271</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel" >
     <x>318</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
